check

```diff
@@ -28,18 +28,30 @@ def get_time(t):
         return int(time.mktime(dt.timetuple()))
 
 
-def checkin(when=None):
+def checkin(when_or_message=None, message=None):
     if db:
         last_checkin = db[-1]
         assert 'out' in last_checkin, "Already checked in"
-    db.append({'in': get_time(when)})
+    try:
+        get_time(when_or_message)
+    except:
+        message = when_or_message
```

Comment:
Wouldn't it be cleaner to do an `is None` check here instead?

Reply:
I really like this small tool :) 👍 
Are you talking about replacing ```try / except``` with if condition ?

Reply:
yes